src/main/java/com/eternalcode/parcellockers/ParcelLockers.java

```diff
@@ -60,22 +58,37 @@ public void onEnable() {
 
         ConfigurationManager configManager = new ConfigurationManager(this.getDataFolder());
         PluginConfiguration config = configManager.load(new PluginConfiguration());
+        if (config.settings.enableSentry) {
+            Sentry.init(options -> {
+                this.getLogger().info("Initializing Sentry...");
+                options.setDsn("https://1dffb5bec4484aaaaca5fcb4c3157a99@o4505014505177088.ingest.sentry.io/4505019784888320");
+                options.setTracesSampleRate(1.0);
+                options.setRelease(this.getDescription().getVersion());
+                options.setTag("serverVersion", this.getServer().getVersion());
+                options.setTag("serverSoftware", PaperLib.getEnvironment().getName());
+                options.setTag("plugins", Arrays.stream(this.getServer().getPluginManager().getPlugins()).toList().toString());
+                this.getLogger().info("Sentry initialized successfully!");
+            });
+        }
```

Comment:
To na pewno takie bezpieczne ustawiać static gościom sentry, i wysyłać do siebie requesty?
Troche syf może się z tym zrobić, np. gdy ktoś wpisze źle passy do bazy danych, bądź coś w configu zwali, raczej takie sentry to jest dla użytkownika żeby mógł monitorować swój serwer w poszukiwaniu, luk i błędów

Reply:
Sentry łapie exceptiony tylko gdy w bloku catch znajdzie się instrukcja _Sentry#captureException_